akka-javasdk/src/main/scala/akka/javasdk/impl/workflow/ReflectiveWorkflowRouter.scala

```diff
@@ -30,35 +32,33 @@ class ReflectiveWorkflowRouter[S, W <: Workflow[S]](
   override def handleCommand(
       commandName: String,
       state: S,
-      command: Any,
+      command: BytesPayload,
       commandContext: CommandContext): Workflow.Effect[_] = {
 
     workflow._internalSetCurrentState(state)
```

Comment:
I think we should have symmetry of `_internalSetCurrentState` and `_internalClearCurrentState`, i.e. that those are called from same method (with clear in a finally). See ESE. Similar with the context if that is also set.